### PR TITLE
test(e2e): xpub test refactor

### PR DESCRIPTION
**Pull request by bosomt**

### CODE REVIEW

File: packages/integration-tests/projects/suite-web/tests/wallet/check-coins-xpub.test.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    const coins: NetworkSymbol[] = ['btc', 'ltc', 'vtc', 'ada'];
+
+    /**
+     * 1. Start at accounts
+     * 2. If not Bitcoin activate coin
+     * 3. apply coin filter
+     * 4. check xpub
+     * 5. x
+     */
```
Comment (ondracja) [inline packages/integration-tests/projects/suite-web/tests/wallet/check-coins-xpub.test.ts:36]: Update test case documentation and remove blank line on 35

File: packages/integration-tests/projects/suite-web/tests/wallet/check-coins-xpub.test.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        cy.prefixedVisit('/accounts');
+        cy.passThroughInitialRun();
+        cy.discoveryShouldFinish();
+    });
+
+    afterEach(() => {
+        cy.task('stopEmu');
+    });
+
+    const coins: NetworkSymbol[] = ['btc', 'ltc', 'vtc', 'ada'];
```
Comment (ondracja) [inline packages/integration-tests/projects/suite-web/tests/wallet/check-coins-xpub.test.ts:26]: refactor this variable to an object that holds `coin` and `xpub identifier`

File: packages/integration-tests/projects/suite-web/tests/wallet/check-coins-xpub.test.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    coins.forEach(coin => {
+        it(`Check ${coin} XPUB`, () => {
+            if (coin !== 'btc') {
+                onAccountsPage.activatNewCoin(coin);
+                cy.getTestElement(`@account-menu/filter/${coin}`).click();
+            }
+
+            cy.getTestElement('@wallet/menu/wallet-details').click();
+            cy.getTestElement('@wallets/details/show-xpub-button').click();
+            cy.getTestElement('@xpub-modal/xpub-field').should('exist');
```
Comment (ondracja) [inline packages/integration-tests/projects/suite-web/tests/wallet/check-coins-xpub.test.ts:46]: add xpub identifier check

**Review by ondracja — CHANGES_REQUESTED** [review]

File: packages/integration-tests/projects/suite-web/tests/wallet/check-coins-xpub.test.ts (hunk 1 above)
Comment (bosomt) [reply packages/integration-tests/projects/suite-web/tests/wallet/check-coins-xpub.test.ts:36]: done

File: packages/integration-tests/projects/suite-web/tests/wallet/check-coins-xpub.test.ts (hunk 2 above)
Comment (bosomt) [reply packages/integration-tests/projects/suite-web/tests/wallet/check-coins-xpub.test.ts:26]: wasnt really needed, all used coins are using zpub prefix

File: packages/integration-tests/projects/suite-web/tests/wallet/check-coins-xpub.test.ts (hunk 3 above)
Comment (bosomt) [reply packages/integration-tests/projects/suite-web/tests/wallet/check-coins-xpub.test.ts:46]: done

**Review by ondracja — APPROVED** [review]

**Pull request merged**